apps/dashboard/app/controllers/transfers_controller.rb

```diff
@@ -17,7 +17,18 @@ def show
 
   def create
     body_params = JSON.parse(request.body.read).symbolize_keys
-    @transfer = Transfer.build(action: body_params[:command], files: body_params[:files])
+
+    from_fs = body_params.fetch(:from_fs, "fs")
+    to_fs = body_params.fetch(:to_fs, "fs")
```

Comment:
I wonder if we want to keep the same pattern of `src` and `dest` instead of `to` and `from`. They're the same semantically, I just wonder if we want to standardize the language everywhere.

Reply:
Using `src` and `dest` sounds better to me. Current naming of `from` and `to` is from `clip_board.js` which uses that naming pattern. Is this something we want to standardize across both the backend and frontend, or would simply renaming the request parameters be sufficient?

Reply:
I think at least changing new code is our best bet. We can keep the js as is and defer it's refactor. 

So, we can't change what we're getting, but we can change what we name the internal variable sort of thing.
```suggestion
    src_fs = body_params.fetch(:from_fs, "fs")
    dest_fs = body_params.fetch(:to_fs, "fs")
```

Reply:
Do we want to change to use `src_fs` and `dest_fs` as the names of the params in the request at the same time too as this PR introduces that? Easier to get that right from the beginning than change it later.
I.e. change only the body here:
https://github.com/OSC/ondemand/blob/bc733533fa94f53440ca97a6d74d8f51b6508e28/apps/dashboard/app/javascript/packs/files/file_ops.js#L473-L481

Reply:
> Easier to get that right from the beginning than change it later.

I tend to think the opposite with changes these large and #2200 possibly conflicting. But if you have the time, you can go ahead and change it - I don't feel too strongly about now or later and #2200 can rebase or merge.